db/schema.rb

```diff
@@ -68,6 +68,7 @@
     t.datetime "updated_at"
     t.integer  "course_id",          limit: 4
     t.string   "display_name",       limit: 255
+    t.integer  "category_id",        limit: 4
```

Comment:
Dang.  I'm thinking the default_scope might be a bad idea, and instead we should have a named scope like `joined`.  This is based on me reading some SO answers that said default_scope is a bad idea.  How many places do we load CUDs, and would it be hard to add a scope to those places?